indexer/packages/postgres/src/models/firebase-notification-token-model.ts

```diff
@@ -0,0 +1,27 @@
+import { Model } from 'objection';
```

Comment:
can this file follow the existing convention of other model files in terms of ordering, `jsonSchema`, and `sqlToJsonConversions`, `tableName`, idColumn`?

Reply:
Still doesn't match other files in term of ordering etc. 

for example, `asset-model.ts` has following structure: ```
export default class AssetModel extends Model {
  static get tableName() {
  }

  static get idColumn() {
  }

  static relationMappings = {};

  static get jsonSchema() {}

  static get sqlToJsonConversions() {}

  id!: string;

  symbol!: string;

  atomicResolution!: number;

  hasMarket!: boolean;

  marketId?: number;
}
```